### PR TITLE
fix(app-activity): Remove underscores from resin feature name

### PR DESCRIPTION
**Pull request by ctuckz**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @ctuckz, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by ctuckz** [comment]
CLA signed

File: src/elements/content-sidebar/activity-feed/app-activity/AppActivity.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ class AppActivity extends React.PureComponent<Props, State> {
                     'bcs-is-pending': isPending || error,
                 })}
                 data-resin-target="loaded"
-                data-resin-feature={`app_activity_${templateId}_card`}
+                data-resin-feature={`appActivityCard${templateId}`}
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/app-activity/AppActivity.js]: Do we still want a delimiter between `appActivityCard` and the value of `templateId`?

Comment (ctuckz) [reply src/elements/content-sidebar/activity-feed/app-activity/AppActivity.js]: Ideally yeah, but we were running into issues ingesting the resin data when there was a delimiter. I moved `templateId` to the end of the string to make it easier to parse without delimiters

**Pull request merged**